reana_db/models.py

```diff
@@ -711,8 +712,12 @@ def _update_cpu_quota(workflow):
         RunStatus.failed,
         RunStatus.stopped,
     ]:
-        _update_cpu_quota(workflow)
-        _update_disk_quota(workflow)
+        quota_update_policy = os.getenv("REANA_WORKFLOW_TERMINATION_QUOTA_UPDATE_POLICY", "cpu,hdd").split(",")
```

Comment:
As we call it `disk`, please change `hdd` to `disk` everywhere

Reply:
Also, it would be good to move `os.getenv()` to `reana_db/config.py` next to the others.

Reply:
BTW the check might be needed in more places. For my tests, I ended up using the following to turn off all cpu and disk quota updating:

```diff
diff --git a/reana_db/models.py b/reana_db/models.py
index c9c007e..87e54ad 100644
--- a/reana_db/models.py
+++ b/reana_db/models.py
@@ -583,6 +583,7 @@ class Workflow(Base, Timestamp, QuotaBase):
 
     def get_workspace_disk_usage(self, summarize=False, search=None):
         """Retrieve disk usage information of a workspace."""
+        return [{"name": "", "size": 0}]
         from functools import partial
 
         if not summarize:
@@ -684,10 +685,12 @@ def workflow_status_change_listener(workflow, new_status, old_status, initiator)
     from .database import Session
 
     def _update_disk_quota(workflow):
+        return
         update_users_disk_quota(user=workflow.owner)
         store_workflow_disk_quota(workflow)
 
     def _update_cpu_quota(workflow):
+        return
         terminated_at = workflow.run_finished_at or workflow.run_stopped_at
         if workflow.run_started_at and terminated_at:
             cpu_time = terminated_at - workflow.run_started_at
diff --git a/reana_db/utils.py b/reana_db/utils.py
index e9ccec7..e96f29c 100644
--- a/reana_db/utils.py
+++ b/reana_db/utils.py
@@ -220,6 +220,7 @@ def update_users_disk_quota(user=None, bytes_to_sum=None):
     :type bytes_to_sum: int
 
     """
+    return
     from reana_commons.utils import get_disk_usage
 
     from reana_db.config import DEFAULT_QUOTA_RESOURCES
@@ -249,6 +250,7 @@ def update_users_disk_quota(user=None, bytes_to_sum=None):
 
 def get_disk_usage_or_zero(workspace_path):
     """Get disk usage for the workspace if exists, zero if not."""
+    return 0
     from reana_commons.utils import get_disk_usage
     from reana_commons.errors import REANAMissingWorkspaceError
 

```